tests/cnf/core/network/dpdk/tests/rootless.go

```diff
@@ -392,14 +406,20 @@ var _ = Describe("rootless", Ordered, Label(tsparams.LabelSuite), ContinueOnFail
 				clientPod.Object.Annotations["k8s.v1.cni.cncf.io/network-status"])
 			Expect(err).ToNot(HaveOccurred(), "Fail to collect PCI addresses")
 
-			rxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapOneInterfaceName, pciAddressList[0]))
+			err = cmd.RxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapOneInterfaceName, pciAddressList[0]))
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
+				"The Receive traffic test on the the client pod failed %s", err))
+
 			checkRxOutputRateForInterfaces(
 				clientPod, map[string]int{
 					tapOneInterfaceName:          minimumExpectedDPDKRate,
 					firstInterfaceBasedOnTapOne:  minimumExpectedDPDKRate,
 					secondInterfaceBasedOnTapOne: maxMulticastNoiseRate,
 				})
-			rxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapTwoInterfaceName, pciAddressList[1]))
+			err = cmd.RxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapTwoInterfaceName, pciAddressList[1]))
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
```

Comment:
Remove fmt.Sprintf, the error will shown anyway

Reply:
removed